### PR TITLE
Updated the OpenEBS cleanup role and added it as part of CI.

### PR DESCRIPTION
**Pull request by yudaykiran**
**What this PR does / why we need it**:
- Updates the OpenEBS cleanup role.
- Updates the stern version to 1.7.0
- Updates the ci.yml to include the cleanup role.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
Signed-off-by: Uday Kiran Y <uday.kiran@openebs.io>

### CODE REVIEW

File: e2e/ansible/roles/k8s-openebs-cleanup/tasks/main.yml [hunk 1]
```diff
@@ -1,32 +1,63 @@
...
+  shell: source ~/.profile; kubectl get sc | grep openebs.io/provisioner-iscsi | cut -d ' ' -f1
+  args:
     executable: /bin/bash
+  register: list_sc
+
+- name: Delete storage classes deployed from other YAMLs
+  shell: source ~/.profile; kubectl delete sc {{ item }}
+  args:
+    executable: /bin/bash
+  with_items: "{{ list_sc.stdout_lines }}"
```
Comment (ksatchit) [inline e2e/ansible/roles/k8s-openebs-cleanup/tasks/main.yml]: :+1:

File: e2e/ansible/cleanup-openebs.yml [hunk 2]
```diff
@@ -2,4 +2,5 @@
 - hosts: kubernetes-kubemasters
   roles:
     - {role: k8s-openebs-uninstall-helm, when: deployment_mode == "hyperconverged" and openebs_deploy == "helm"}
+    - {role: k8s-openebs-cleanup}
```
Comment (ksatchit) [inline e2e/ansible/cleanup-openebs.yml]: Does this need a conditional invocation when openebs_deploy=kubectl ?

Comment (yudaykiran) [reply e2e/ansible/cleanup-openebs.yml]: You are right, we will need it.

File: e2e/ansible/roles/k8s-openebs-cleanup/tasks/main.yml [hunk 3]
```diff
@@ -1,32 +1,63 @@
...
     executable: /bin/bash
 
 - name: Confirm pod has been deleted
-  shell: source ~/.profile; kubectl get pods 
+  shell: source ~/.profile; kubectl get pods -n openebs
   args:
     executable: /bin/bash
   register: result
-  until: "'maya-apiserver' or 'openebs-provisioner' not in result.stdout"
+  until: "'maya-apiserver' or 'openebs-provisioner or 'openebs-snapshot-operator' not in result.stdout"
```
Comment (kmova) [inline e2e/ansible/roles/k8s-openebs-cleanup/tasks/main.yml]: _n, a ' is missing after openebs-provisioner .

Comment (yudaykiran) [reply e2e/ansible/roles/k8s-openebs-cleanup/tasks/main.yml]: Thanks, fixed it.

**Pull request merged**